lib/archiving.js

```diff
@@ -242,6 +243,7 @@ exports.startArchive = function (config, sessionId, options, callback) {
     hasAudio: options.hasAudio,
     hasVideo: options.hasVideo,
     outputMode: options.outputMode,
+    layout: options.layout,
```

Comment:
Hmm, please correct me if I'm wrong, but I don't see `layout` as a part of the properties for the API call: https://tokbox.com/developer/rest/#start_archive

Reply:
The `layout` property is missing from the REST API documentation. However, it is mentioned in this developer guide: https://tokbox.com/developer/guides/archiving/layout-control.html#specifying-the-initial-layout-type.

I have a note to add the missing info to the REST API docs.